lib/manager/components/version/FeedVersionTabs.js

```diff
@@ -107,8 +106,8 @@ export default class FeedVersionTabs extends Component<Props> {
             <Row>
               {countFields.map(c => {
                 const summary = version.validationSummary
-                const diff = comparedVersion ? (summary[c] - comparedVersion.validationSummary[c]) : 0
-                const value = summary[c]
+                const diff = comparedVersion ? ((summary[c] || 0) - (comparedVersion.validationSummary[c] || 0)) : 0
+                const value = summary[c] || 0
```

Comment:
Maybe swap these two rows and reuse variable `value`.

Reply:
Good catch! Refactored in 15a6f1ad6aa68131ffe4fc4b026bef670d2b1eac